### ISSUE TITLE
postgresql-rc.yaml: psql: FATAL:  database "postgres" does not exist

### ISSUE THREAD

**Issue by alogoc** [sameersbn/docker-gitlab#846]
I attempted to deploy this on kubernetes using the latest yaml provided from [https://github.com/sameersbn/docker-gitlab/tree/master/kubernetes](url) however the postgresql pod is failing to start.

Error: 

```
kubectl  --namespace=gitlab log postgresql-4w83r
Initializing datadir...
Initializing certdir...
Initializing logdir...
Initializing rundir...
Setting resolv.conf ACLs...
Creating database user: gitlab
psql: FATAL:  database "postgres" does not exist
psql: FATAL:  database "postgres" does not exist
```

**Comment by SolidNerd** [comment]
Hey,
I have currently no clue where the error comes from. @sameersbn any idea ?

**Comment by alogoc** [comment]
Looks like this specific issue was solved by the newer sameersbn/postgresql:9.5-1 image.

Closing the issue.
